cmd/kubelet/app/server.go

```diff
@@ -314,9 +313,6 @@ func run(s *options.KubeletServer, kcfg *KubeletConfig) (err error) {
 	if s.ExitOnLockContention && s.LockFilePath == "" {
 		return errors.New("cannot exit on lock file contention: no lock file specified")
 	}
-	if err := checkPermissions(); err != nil {
```

Comment:
Just to get the error message to be below the kubeconfig message, so that if we are in a crash loop we don't print that message before we exit.